pkg/cloud/services/securitygroup/securitygroups.go

```diff
@@ -363,6 +368,53 @@ func (s *Service) describeSecurityGroupsByName() (map[string]infrav1.SecurityGro
 	return res, nil
 }
 
+// revokeIngressAndEgressRulesFromVPCDefaultSecurityGroup revokes ingress and egress rules from the VPC default security group.
+// The VPC default security group is created by AWS and cannot be deleted.
+// But we can revoke all ingress and egress rules from it to make it more secure. This security group is not used by CAPA.
+func (s *Service) revokeIngressAndEgressRulesFromVPCDefaultSecurityGroup() error {
+	if s.scope.VPC().SecureDefaultVPCSecurityGroup {
```

Comment:
Please return early instead of indenting the whole function code. Or alternatively, you could move the condition to the caller.

Reply:
Done